src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -44,7 +44,15 @@ class FilterOptionsConfig {
     this.optionType = config.optionType || OptionTypes.STATIC_FILTER;
 
     /**
-     * The list of filter options to display with checked status
+     * The list of filter options to display with checked status as
+     * initially specified in the user configuration
+     * @type {object[]}
+     */
+    this.initialOptions = config.options.map(o => ({ ...o }));
+
+    /**
+     * The list of filter options to display with checked status,
+     * changing with changes in the persistent storage.
```

Comment:
Is there a way to rephrase "changing with changes in the persistent storage"? I think this might confuse someone because they'll see this comment but that it's getting modified in `globalStorage` update listener.

Reply:
Terribly worded haha, I removed it, I think it's not worth mentioning when an instance variable is dynamic